demonstrations/tutorial_falqon.py

```diff
@@ -0,0 +1,452 @@
+r"""
+Feedback-Based Quantum Optimization (FALQON)
+=========================
+
+.. meta::
+    :property="og:description": Solve combinatorial problems with no classical optimizer
+    :property="og:image": https://pennylane.ai/qml/_images/qaoa_layer.png
+
+.. related::
+
+   tutorial_qaoa_intro Intro to QAOA
+   tutorial_qaoa_maxcut QAOA for MaxCut
+
+*Authors: David Wakeham and Jack Ceroni. Posted: XXX. Last updated: XXX.*
+
+-----------------------------
+
+While the `Quantum Approximate Optimization Algorithm (QAOA) <https://pennylane.ai/qml/demos/tutorial_qaoa_intro.html>`__
+is one of the best-known processes for solving combinatorial optimization problems with quantum computers,
+it has drawbacks. Since QAOA relies on optimization over a parameter space,
+convergence isn't guaranteed, and the optimization procedure can become "stuck" in local minima.
```

Comment:
```suggestion
it has drawbacks: convergence isn't guaranteed, and the optimization procedure can become
"stuck" in local minima.
```
It's not accurate that optimization over a parameter space (is there any other typo of optimization?) implies that convergence isn't guaranteed